postgresql-16.yaml

```diff
@@ -71,7 +83,16 @@ pipeline:
         --with-openssl \
         --with-libedit-preferred \
         --with-uuid=e2fs \
-        --with-libxml
+        --with-libxml \
+        --with-libxslt \
+        --with-icu \
+        --with-lz4 \
+        --with-zstd \
+        --with-gssapi \
+        --with-ldap \
+        --with-llvm \
+        --enable-thread-safety \
+        --with-system-tzdata=/usr/share/zoneinfo
```

Comment:
`tzdata` will need to be bundled as a runtime dependency

Reply:
Thanks, RJ, pushed a change to that end